### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by nschonni**
Copied from Ruby-Sass

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Sass is more than a technology; Sass is driven by the community of individuals
+that power its development and use every day. As a community, we want to embrace
+the very differences that have made our collaboration so powerful, and work
+together to provide the best environment for learning, growing, and sharing of
+ideas. It is imperative that we keep Sass a fun, welcoming, challenging, and
+fair place to play.
+
+[The full community guidelines can be found on the Sass website.][link]
+
+[link]: http://sass-lang.com/community-guidelines
```
Comment (asottile) [inline CODE_OF_CONDUCT.md]: should this be `https`?

Comment (nschonni) [reply CODE_OF_CONDUCT.md]: Probably 😝 but I just did the same PR to the other repos in the org like this. I'm going to check to see if they're going to set up the HTTPS redirect on the site

Comment (asottile) [reply CODE_OF_CONDUCT.md]: :rip: time to fix them all 😆

Comment (asottile) [reply CODE_OF_CONDUCT.md]: (just in the sass org):
```console
$ all-repos-grep -C all-repos-sass.json http://sass-lang.com | wc -l
753
$ all-repos-grep -C all-repos-sass.json https://sass-lang.com | wc -l
9
$ all-repos-grep -C all-repos-sass.json http://sass-lang.com | cut -d':' -f1 | sort | uniq -c | sort -rn
    684 output-sass/sass/sass-spec
     23 output-sass/sass/ruby-sass
     15 output-sass/sass/sass
      5 output-sass/sass/sass-site
      5 output-sass/sass/node-sass
      5 output-sass/sass/libsass
      4 output-sass/sass/sass-registry
      4 output-sass/sass/dart-sass
      2 output-sass/sass/sassc
      2 output-sass/sass/perl-libsass
      2 output-sass/sass/language
      1 output-sass/sass/node-sass-middleware
      1 output-sass/sass/libsass.com
```

lmk and I can PR fixes for them with a [single command](https://github.com/asottile/all-repos#all-repos-sed-options-expression-filenames)

**Pull request merged**